hysr/scene.py

```diff
@@ -0,0 +1,41 @@
+import typing
+
+from .defaults import Defaults
+
+
+class Pos:
```

Comment:
Maybe rename this class to "Pose".  I assumed "Pos" to be short for "Position" and thus didn't expect it to contain an orientation.